CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+Guidelines on how to contribute to KubeSlice
+
+There are a number of ways you can contribute to the KubeSlice project. We would like to accept your patches and contributions to the KubeSlice code base. This document provides guidelines on how to contribute to the KubeSlice code base. we require all participants to adhere to the [Code of Conduct](https://github.com/kubeslice/worker-operator/blob/doc/contributing-guide/code_of_conduct.md).
+
+
+Here are some ways to contribute:
+
+* Contribute to the kubeslice codebase
+* Report and triage bugs.
+* Answer questions on KubeSlice discussions forums
+* Write technical documentation and blog posts, for users and contributors.
+* Filing issues
+* Provide feedback
+* Submit PRs for small bug fixes
+
+## Contributor Compliance with Developer Certificate Of Origin (DCO)
+
+We require every contributor to certify that they are legally permitted to contribute to our project. A contributor expresses this by consciously signing their commits, and by this act expressing that they comply with the Developer Certificate Of Origin
+
+A signed commit is a commit where the commit message contains the following content:
+
+`Signed-off-by: John Doe <jdoe@example.org>`
+
+This can be done by adding `--signoff` to your git command line
+
+## Code of Conduct
+Please make sure to read and observe our [Code of Conduct](https://github.com/kubeslice/worker-operator/blob/doc/contributing-guide/code_of_conduct.md).
+
+## Contributor Workflow
+
+Pull Requests are welcome. In general, we follow the "fork-and-pull" Git workflow.
```

Comment:
`In general, we follow the **fork-and-pull** Git workflow that has the following steps: ` 

This sentence follows a list; so change the sentence that starts with `In general...` as above and end with a colon.